ci/ansible/roles/pulp-smash/defaults/main.yml

```diff
@@ -9,3 +9,9 @@ pulp_smash_password: "admin"
 
 # If SSL certificate should be verified
 pulp_smash_verify: "false"
+
+# Pulp Server version
+# pulp_smash_version variable is optional, if you want to specify it then you
+# can do so by passing additional variables to ansible-playbook command or by
+# uncommenting the following line and editing as needed.
+# pulp_smash_version: 2.7
```

Comment:
Well, the defaults are there so that you've got a default value, so commenting it out is confusing to me.

Wouldn't the version of pulp you're testing against be something you can get from a custom fact?